### PR TITLE
Awesome books: 

### PR DESCRIPTION
**Pull request by X-Elie-X**
Elie contributions
-Add hovers  effect.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -12,30 +14,30 @@ class store {
...
 
 class ui {
   static displayBooksCollection() {
     const booksCollection = store.getbook();
     const booklist = document.querySelector('.bookList');
-    booksCollection.forEach(book => {       
+    booksCollection.forEach((book) => {
       const bookFrame = document.createElement('div');
       bookFrame.classList.add('bookFrame');
       bookFrame.innerHTML = `
```
Comment (thecodechaser) [inline main.js:43]: - [ ] The book list and the add book form should be on the same page as on the wireframe. Please fix this issue and make sure that your design follows the wireframe design.

### Your design

image

### Wireframe design 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ChaabaneAnas, and @X-Elie-X👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @ChaabaneAnas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️
- No linter errors
- Working project

## Highlights

## Optional suggestions

Consider putting some space between the author's name and the 'by' word.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (ChaabaneAnas) [reply main.js:43]: Hello @thecodechaser,
can you explain what exactly do you require?
i mean is it just the border?

Comment (thecodechaser) [reply main.js:43]: @ChaabaneAnas, It's not about the borders. For now, the book list and the add from are in two different sections. The form should be placed below the boom list.

Comment (ChaabaneAnas) [reply main.js:43]: @thecodechaser 
oh I see,
this actually  the finished version of the project  please see the screenshot below
Screenshot from 2022-08-01 10-16-36

Comment (thecodechaser) [reply main.js:43]: @ChaabaneAnas, I understand that but as code reviewers, we have some guidelines to follow.

Comment (ChaabaneAnas) [reply main.js:43]: @thecodechaser, that doesn't make sense, 
when you require me to actually remove functionality,
 instead of adding one ...

Comment (thecodechaser) [reply main.js:43]: @ChaabaneAnas, When we perform code review, We have some guidelines for code and design. We strictly need to follow them that is what we're getting paid for. I understand your case is different. But we need to stick to the guidelines if we do not want to lose our job. 😄

Comment (ChaabaneAnas) [reply main.js:43]: @thecodechaser, Yeah i think ur right im removing the functionality

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @X-Elie-X and @ChaabaneAnas,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**